src/main/java/org/ibp/api/rest/germplasm/GermplasmAttributeResource.java

```diff
@@ -0,0 +1,84 @@
+package org.ibp.api.rest.germplasm;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.generationcp.middleware.api.attribute.AttributeDTO;
+import org.generationcp.middleware.domain.germplasm.GermplasmAttributeDto;
+import org.generationcp.middleware.domain.germplasm.GermplasmAttributeRequestDto;
+import org.ibp.api.java.germplasm.GermplasmAttributeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.List;
+import java.util.Set;
+
+@Api(value = "Germplasm Attribute Services")
+@Controller
+public class GermplasmAttributeResource {
+
+	@Autowired
+	private GermplasmAttributeService germplasmAttributeService;
+
+	@ApiOperation(value = "Returns germplasm attributes filtered by a list of codes and attibute type",
```

Comment:
I think this is not the right place for this service but in the other hand, attributes will be migrated to the ontology so we will need to work on this later on

Reply:
Okay. I'll leave it as is for now.